sumo-logic/release_notes.md

```diff
@@ -1,5 +1,5 @@
 ## What's Fixed
 ### Added Operations:
-- Retrieve the List of Insights by Query
-- Retrieve the List of All Insights
-- Retrieve Details by Insights ID
\ No newline at end of file
+- Get the List of Insights By Query
+- Get the List of All Insights
+- Get Details By Insights ID
```

Comment:
update the action names
- Get Insights List By Query
- Get All Insights List
- Get Insight Details By ID